app/controllers/review_bids_controller.rb

```diff
@@ -96,26 +103,49 @@ def set_priority
   def assign_bidding
```

Comment:
What in the world does "assign_bidding" mean?  Does it mean "assign_reviews" to participants who have bid?  This method needs a better name.